src/openrct2/actions/GameActionRegistration.cpp

```diff
@@ -138,6 +140,7 @@ namespace GameActions
         Register<LandLowerAction>();
         Register<LandRaiseAction>();
         Register<LandSetHeightAction>();
+        Register<LandSetRightsAction>();
         Register<LandSmoothAction>();
```

Comment:
Missing LandBuyRightsAction registration

Reply:
I think its when I rebase that it sometimes deletes my register as I've tested these a few times. Thanks for noticing.